common/src/main/java/gov/cms/ab2d/common/dto/StartJobDTO.java

```diff
@@ -0,0 +1,21 @@
+package gov.cms.ab2d.common.dto;
+
+import gov.cms.ab2d.fhir.FhirVersion;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Data //NOSONAR
+@AllArgsConstructor
+public class StartJobDTO {
+    private final String contractNumber;
+    private final String organization;  //NOSONAR - not quite in use yet
```

Comment:
Is this needed now that lombok is not covered by sonar

Reply:
Yeah, I'll clean that up in the next round.